### ISSUE TITLE
The fan starts only at high temperatures

### ISSUE THREAD

**Issue by smartpie** [crystalidea/macs-fan-control#780]
Until the temperatures of the Mac rise to those values at which the Mac decides that it is time for him to turn on the fans, it will not be possible to start the fans through the application. But at the same time, when the temperature is already high enough and the mac decides to start the fans on its own, then using the application it is already possible to control how fast the fans will work That is, the rotation speed control works, but not the fact that the fans are on or off

MacBook Pro, 16-inch, Nov 2023, M3 Pro, Sonoma 14.5

*On my older MacBookPro 13-inch on M1 everything worked and works well on all version of Mac OS (that was avaible on arm based macs)

**Comment by snougo** [comment]
same issue on M3 Max

**Comment by ispaydeu** [comment]
Also have this issue on M3 Macbook Max Pro 128 GB. Macbook will get really hot, fans sit at 0 RPM, then suddenly they go to full blast full RPM. I paid for the full version hoping that would fix it and no luck. Please fix this.

**Comment by cigsz** [comment]
This is not Macs fan control app's fault. 
Apple has blocked to manually control fans when they're off. It looks like they changed something with M3 and probably M4 Macs.
I have tried 3 different apps - Macs Fan Control, TG Pro and iStat and all of them don't work. 
I have even received confirmation from TG Pro's developer that Apple has changed how fans operate and when the fans turn on. It probably won't be possible to turn on fans until Apple make a difference and bring back how they used to work.
I have filed several feedbacks and under one Apple responded that it "works as designed".
So people need to show Apple that they want the control of fans back. Maybe then they'll revert those changes.
